week05/errors_test.go

```diff
@@ -0,0 +1,92 @@
+package demo
+
+import (
+	"fmt"
+	"testing"
+)
+
+func Test_tablenotfound(t *testing.T) {
+	t.Parallel()
+
+	tn := "users"
+
+	errTable := ErrTableNotFound{
+		table: tn,
+	}
+
+	if errTable.Error() == "" {
+		t.Fatalf("table %s not found", errTable.table)
+	}
```

Comment:
This test just makes sure that the error message isn't an empty string. It doesn't assert that the error message is properly laid out.

```suggestion
        exp := fmt.Sprintf("table %s not found", tn)
        act := errTable.Error()
	if act != exp {
		t.Fatalf("expected %q, got %q", exp, act)
	}
```